docs/20_tutorial.md

```diff
@@ -1,64 +1,245 @@
 \pagebreak
 
-Run Book outline
-========
+# Run Book 
 
-***should the run book deal with the SOS establishing the risk
-limit, or any of their pre-audit responsibilities like the selection
-of which races to audit?
+## Introduction
 
-Brief introduction to RLAs in non-technical, human friendly language.
-At the county level, something along the lines of "This tool is going
-to help you cross reference votes recorded on paper ballots with how 
-the computers read those ballots, to determine whether the computers
-did their job accurately…” Try to put their minds at ease about the
-process. :)
+### What the RLA Tool Does
+
+The Risk-Limiting Audit (RLA) tool, developed by Free & Fair 
+for the Colorado Department of
+State for use in elections from November 2017 forward, supports a 
+Risk-Limiting Audit as required by Colorado statute and 
+as described in the Colorado Secretary of State's Rule 25 for
+elections. RLA-related terms and processes are defined and prescribed in Rule 25.
+*** citation for Rule 25.
+
+The RLA Tool enables the Secretary of State and each individual 
+County to carry out a Risk-Limiting Audit. The Secretary of State can
+enter the risk limit, the random seed and the contests that drive
+the audit process. The Secretary of State can also identify 
+contests that should go to a full hand count. Counties can upload
```

Comment:
That sounds like the right contrast. But this is the intro of the document that people will be more likely to read, and I just don't want to scare them off at the beginning with talk of full hand counts. So I'm just suggesting that we could say here in the intro something like "and all other necessary functions", and leave the full hand count details for the appropriate places.